### ISSUE TITLE
Compile error in master branch - should be Game_mode instead of Game_Mode?

### ISSUE THREAD

**Issue by kreatordxx** [dxx-rebirth/dxx-rebirth#267]
When I compiled **master** (using Apple's Clang) I noticed this compiler error:
```
CXX d1x . similar/main/object.cpp
similar/main/object.cpp:832:7: error: use of undeclared identifier 'Game_Mode'; did you mean 'Game_mode'?
        if (Game_Mode & GM_MULTI)
            ^~~~~~~~~
            Game_mode
common/main/game.h:101:12: note: 'Game_mode' declared here
extern int Game_mode;
           ^
1 error generated.
scons: *** [similar/main/.d1x-rebirth.object.o] Error 1
scons: building terminated because of errors.

```

**Comment by Mako88** [comment]
This was mentioned near the bottom of #249, and should be fixed once the patch @vLKp suggested is committed.

**Comment by vLKp** [comment]
As Mako says, I [posted a fix for this in issue #249](249#issuecomment-259588339), pending review from zico.  He subsequently submitted a change that resolves this build failure.  His change is different from the one I proposed.
